lib/css-assembler.js

```diff
@@ -44,6 +44,7 @@ function assemble(cssFiles, absolutePath, type, options, callback) {
     });
 
     function parseImports(cssFileAlias, callback) {
+        cssFileAlias = cssFileAlias.replace(/(\.scss)/g, '');
```

Comment:
It might be worth anchoring this to the end of the filename, i.e. `/(\.scss)$/g`.

Reply:
Thats a good point. Updated in new commit.